CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+## Contributing
+
+[fork]: /fork
+[pr]: /compare
+[style]: https://standardjs.com/
+[code-of-conduct]: CODE_OF_CONDUCT.md
+
+Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+Please note that this project is released with a [Contributor Code of Conduct][code-of-conduct]. By participating in this project you agree to abide by its terms.
+
+## Issues and PRs
+
+If you have suggestions for how this project could be improved, or want to report a bug, open an issue! We'd love all and any contributions. If you have questions, too, we'd love to hear them.
+
+We'd also love PRs. If you're thinking of a large PR, we advise opening up an issue first to talk about it, though! Look at the links below if you're not sure how to open a PR.
+
+Work in Progress pull requests are also welcome to get feedback early on, or if there is something blocked you.
+
+## Submitting a pull request
+
+1. [Fork][fork] and clone the repository.
+1. Create a new branch: `git checkout -b my-branch-name`.
+1. Configure and install the dependencies: `poetry install`.
+1. Make sure the tests pass on your machine: `make install & make tests`
```

Comment:
A migration to `task` should be considered. `make` is not really a build tool the python community use.

Reply:
apologies for the confusion, i removed these files as they are really part of #681

Reply:
@jerome-benoit which `task` library were you referring to? there's pytask. but also snakemake, scons, invoke, bitbake all of which are python friendly, although probably way more functionality than is needed here.

Reply:
`taskpy`. The `Makefile` is only used here to run ordered shell command(s), so is pointless.  `taskpy` can do the same without requiring another external tool.